### PR TITLE
update RTMA to v0.7.2 and v0.7.3

### PR DESCRIPTION
**Pull request by guoqing-noaa**
## DESCRIPTION OF CHANGES: 

- update RTMA to v0.7.2 and v0.7.3

## TESTS CONDUCTED: 

- On machines/platforms:

- [ ] WCOSS2
- [ ] Hera
- [ ] Orion
- [x] Jet

- Test cases: 

- [x] Non-DA engineering test
- [ ] DA engineering test
- [x ] Other sample scripts:  RTMA realtime runs

## ISSUE: 

- Fixes the issue(s) mentioned in #112 

## CONTRIBUTORS (optional): 

@AnnetteGibbs-NOAA

### CODE REVIEW

**Review by EdwardColon-NOAA — APPROVED** [review]

**Comment by MatthewPyle-NOAA** [comment]
Request was made to create a tag once this PR is merged.

File: scripts/exrrfs_run_analysis.sh
Comment (MatthewPyle-NOAA) [inline scripts/exrrfs_run_analysis.sh:1]: Will this copying of the diag_* file to COMOUT impact other applications?   Just was curious about it.

**Review by MatthewPyle-NOAA — COMMENTED** [review]
Just one small question - should be easy to merge and tag today.

File: scripts/exrrfs_run_analysis.sh
Comment (guoqing-noaa) [reply scripts/exrrfs_run_analysis.sh:1]: No, this line only runs when one sets "binary_diag" to true. Right now, only RTMA applications turn on this switch.

Actually, this line is a bug fix as binary diag files should be copied to the com/ directory if turned on, just similar to the netcdf_diag situation.

**Comment by guoqing-noaa** [comment]
> Request was made to create a tag once this PR is merged.

Thanks, @MatthewPyle-NOAA !

**Review by MatthewPyle-NOAA — APPROVED** [review]
Seems good to me.  Will get it merged and tagged.

**Pull request merged**